invenio_rdm_records/services/schemas/metadata.py

```diff
@@ -78,6 +75,21 @@ class CreatorSchema(Schema):
     identifiers = fields.Dict()
     affiliations = fields.List(fields.Nested(AffiliationSchema))
 
+    @post_load
+    def fill_given_family_names(self, data, **kwargs):
+        """Fill the given_name and family_name based on name.
+
+        Allows to override conversion of name to family name / given name
+        if given_name or family_name is explicitly passed.
+        """
+        if data["type"] == "personal":
+            family_name, given_name = (
+                [n.strip() for n in data["name"].partition(",")][0::2]
```

Comment:
This will not work in general. For personal names, we need people to explicitly specify the two fields, family name and given name. The only thing we can do is the reverse - i.e. ``name = f"{family_name}, {given_name}"``. 

Even though technically, you can tell people this is how it works, then they won't do it correctly. Zenodo has all the time, had a recommendation to enter the information correctly, but most people just copy/paste whatever name they already have written somewhere.

Reply:
Then what I will do is:

- Frontend: 
 * When "organizational" is chosen, show 1 name input field
 * When "personal" is chosen, show given name and last name input fields (only - don't show name input field)

- Backend:
 * When "personal": construct name from last name and first name
 * When "organizational": empty out first name and last name.

Reply:
Ready for re-review!